kotak-website/src/home/App.js

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import './App.css';
+
+class Image extends Component{
+  render() {
+    const styles = {width: '200px', height:'200px', display:'inline-block'}
+    return (
+      <div className="inner_div">
+        <img style={styles} src="favicon.png"></img>
+      </div>
+    )
+  }
+}
+
+class Bio extends Component{
+  render() {
+    return (
+      <div className="inner_div">
+        <h2>Shyam Kotak</h2>
+        <h3>Software Developer @ BrightEdge</h3>
+      </div>
+    )
+  }
+}
+
+function Icon(props){
+  const icon = props.icon[0]
+  const link = props.icon[1]
+  const styles_black = {'color':'black'}
+  const styles = {'color':props.icon[2]}
+  return (
+    <a href={link} target="_blank" rel="noopener noreferrer">
+      <span className="fa-stack fa-2x">
+        <FontAwesomeIcon style={styles_black} className="fa-stack-2x" icon={["far", "circle"]}/>
+        <FontAwesomeIcon style={styles} className="fa-stack-1x" icon={icon}/>
+      </span>
+    </a>
+  );
+}
+
+class IconList extends Component{
+  render() {
+    const icons = [[["fas", "envelope"], "mailto:me@shyamkotak.com", "#000000"],
+                   [["fab", "twitter"], "https://twitter.com/shyamkotak_", "#1da1f2"],
+                   [["fab", "github"], "https://github.com/shyamkotak", "#333"],
+                   [["fab", "linkedin"], "https://www.linkedin.com/in/kotakshyam", "#0077B5"],
+                   [["far", "file-alt"], "Kotak_Resume.pdf", "#000000"]];
+    var list = [];
+    for (var i = 0; i < icons.length; i++) {
```

Comment:
I like how you're rendering your icon list with this for loop. Alternatively, you could use `Array.map()`

This is a classic use-case for a stateful component. Icons can go inside of state and you can do fancy stuff with it like add a hover property to each (false by default) and change it to true on hover and hence change the color of icon on hover or something like that.

Then instead of passing the entire array of an icon you can pass in props like name, link, color, isHover, etc. 

Hope that makes sense.